npm/private/npm_translate_lock.bzl

```diff
@@ -98,8 +99,14 @@ def _npm_translate_lock_impl(rctx):
         if state.action_cache_miss():
             _fail_if_frozen_pnpm_lock(rctx, state)
             if _update_pnpm_lock(rctx, state):
-                # If the pnpm lock file was changed then reload it before translation
-                state.reload_lockfile()
+                if rctx.attr.bzlmod:
+                    msg = """
+
+INFO: {} file updated. Please run your build again.""".format(state.label_store.relative_path("pnpm_lock"))
```

Comment:
nit: I'd suggest a TODO here, or maybe include a link to the issue in the message we print, I hope this is a short-term improvement